ports/esp32/boards/GARATRONIC_PYBSTICK26_ESP32C3/modules/pybstick26_esp32c3.py

```diff
@@ -0,0 +1,29 @@
+# PYBSTICK26-ESP32C3 MicroPython Helper Library
+# V1.0 by fpie for GARATRONIC (2025)
+#
+
+from micropython import const
+from machine import Pin
+
+# Pin Assignments
+
+# SPI
+SPI_MOSI = const(7)
+SPI_MISO = const(2)
+SPI_CLK = const(6)
```

Comment:
You shouldn't really need to add any of these pin aliases.  Instead use a `pins.csv` file (see eg `ports/esp32/boards/UM_TINYC6/pins.csv`).  You could do the following:
```csv
SPI_MOSI,GPIO7
SPI_MISO,GPIO2
SPI_CLK,GPIO6

I2C_SDA,GPIO0
I2C_SCL,GPIO1

LED,GPIO8

BUTTON_A,GPIO9
BUTTON_B,GPIO5
```
You can also add other things like `IO6,GPIO6`.  Generally you want to add all aliases for pins that are labelled on the silk screen, and use the silk screen name for the alias.  That way someone looking at the board can know which name to use for each pin.

Then you could do:
```py
led = Pin("LED", Pin.OUT, value=0)
button = Pin("BUTTON_B", Pin.IN, Pin.PULL_UP)
```

Reply:
Yes. Of course. I take the wrong example with LOLIN_C3_MINI and follow it as a mindless.
Thanks for your help.